[BUG] Failure to build

Fresh clone of the repo fails to build

```
pnpm build

> meshtastic-web@2.3.3-0 build /Users/james/projects-personal/test
> tsc && vite build

src/validation/config/position.ts:5:14 - error TS2420: Class 'PositionValidation' incorrectly implements interface 'Omit<Config_PositionConfig, keyof Message<AnyMessage>>'.
  Type 'PositionValidation' is missing the following properties from type 'Omit<Config_PositionConfig, keyof Message<AnyMessage>>': gpsEnabled, gpsAttemptTime

5 export class PositionValidation
               ~~~~~~~~~~~~~~~~~~


Found 1 error in src/validation/config/position.ts:5

 ELIFECYCLE  Command failed with exit code 2.
```

--- a/src/validation/config/position.ts
+++ b/src/validation/config/position.ts
@@ -7,12 +7,6 @@ const DeprecatedPositionValidationFields = ['gpsEnabled', 'gpsAttemptTime'];
 export class PositionValidation
   implements Omit<Protobuf.Config.Config_PositionConfig, keyof Message | typeof DeprecatedPositionValidationFields[number]>
 {
-  @IsBoolean()
-  gpsEnabled: boolean;
-
-  @IsInt()
-  gpsAttemptTime: number;
-
   @IsInt()
   positionBroadcastSecs: number;
 

--- a/src/validation/config/position.ts
+++ b/src/validation/config/position.ts
@@ -40,4 +40,10 @@ export class PositionValidation
 
   @IsArray()
   channelPrecision: number[];
+
+  @IsInt()
+  gpsAttemptTime: number;
+
+  @IsBoolean()
+  gpsEnabled: boolean;
 }

--- a/src/validation/config/position.ts
+++ b/src/validation/config/position.ts
@@ -2,8 +2,11 @@ import type { Message } from "@bufbuild/protobuf";
 import { Protobuf } from "@meshtastic/js";
 import { IsArray, IsBoolean, IsEnum, IsInt } from "class-validator";
 
+//type DeprecatedPositionValidationFields = 'gpsEnabled' | 'gpdAttempttime';
+const DeprecatedPositionValidationFields = ['gpsEnabled', 'gpsAttemptTime'];
+
 export class PositionValidation
-  implements Omit<Protobuf.Config.Config_PositionConfig, keyof Message>
+  implements Omit<Protobuf.Config.Config_PositionConfig, keyof Message | typeof DeprecatedPositionValidationFields[number]>
 {
   @IsInt()
   positionBroadcastSecs: number;
@@ -40,10 +43,4 @@ export class PositionValidation
 
   @IsArray()
   channelPrecision: number[];
-
-  @IsInt()
-  gpsAttemptTime: number;
-
-  @IsBoolean()
-  gpsEnabled: boolean;
 }

--- a/src/validation/config/position.ts
+++ b/src/validation/config/position.ts
@@ -2,7 +2,6 @@ import type { Message } from "@bufbuild/protobuf";
 import { Protobuf } from "@meshtastic/js";
 import { IsArray, IsBoolean, IsEnum, IsInt } from "class-validator";
 
-//type DeprecatedPositionValidationFields = 'gpsEnabled' | 'gpdAttempttime';
 const DeprecatedPositionValidationFields = ['gpsEnabled', 'gpsAttemptTime'];
 
 export class PositionValidation

